src/global_styling/variables/_typography.ts

```diff
@@ -69,7 +69,7 @@ const font: EuiFont = {
 
 // Font weights
 const fontWeight = {
-  light: '300',
+  weightLight: '300', // TODO
```

Comment:
Yeah so we talked about maybe for the color mode keys, they're uppercased like `LIGHT` and `DARK` leaving any lowercase versions allowed as normal keys here. But I'm fine with this for now.